google/cloud/bigtable/rpc_retry_policy.h

```diff
@@ -158,8 +161,8 @@ class LimitedTimeRetryPolicy : public RPCRetryPolicy {
   bool OnFailure(grpc::Status const& status) override;
 
  private:
-  using Impl =
-      google::cloud::internal::LimitedTimeRetryPolicy<internal::SafeGrpcRetry>;
+  using Impl = google::cloud::internal::LimitedTimeRetryPolicy<
```

Comment:
oops, I should check for `cloud::internal`'s

Reply:
nvm, I think we want to fully qualify this. It was probably missed because of the line break.